torchprime/torch_xla_models/tests/test_llama.py

```diff
@@ -65,11 +65,11 @@ def test_forward_our_model_against_hf_model(self):
       )
       torch_xla.sync()
       self.assertTrue(
-        torch.allclose(hf_output.logits, llama_output.logits, atol=1e-6),
+        torch.allclose(hf_output.logits, llama_output[0], atol=1e-6),
```

Comment:
nit: for readability, we could have something like

```
llama_logits, llama_loss =  model_xla(
  input, labels=input, attention_mask=torch.ones_like(input)
)
```

then we can put `llama_logits` here